script/dump_db

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+cd $DEPLOYMENT_DIR
+sudo -u postgres pg_dump -U postgres -d $PG_SIMPLECMS_PROD >"dump/$(date).dump"
```

Comment:
Assuming everything is occurring 100% on a single machine, this should be a better design. Plus, the script can be modified rather easily.